src/modules/lua_mtev.c

```diff
@@ -2720,7 +2723,7 @@ nl_conf_get_string_list(lua_State *L) {
   const char *base_path = lua_tostring(L,1);
   const char *child_path = lua_tostring(L,2);
 
-  mtev_conf_section_t* mqs = mtev_conf_get_sections(NULL, base_path, &cnt);
+  mtev_conf_section_t* mqs = mtev_conf_get_sections(MTEV_CONF_ROOT, base_path, &cnt);
```

Comment:
looks like we could leave due to an error on line 2736 without calling `mtev_conf_release_sections()`